table/tables/tables.go

```diff
@@ -228,15 +228,16 @@ func (t *Table) UpdateRecord(ctx sessionctx.Context, h int64, oldData, newData [
 	if err != nil {
 		return errors.Trace(err)
 	}
+	numCols := len(newData) + 1
```

Comment:
Please add comments about why we need to add 1. 
I'm afraid we will forget why.